src/gt4py/utils/meta.py

```diff
@@ -535,6 +535,57 @@ def visit_Name(self, node: ast.Name):
 map_symbol_names = SymbolsNameMapper.apply
 
 
+class InsideCall(ASTPass):
```

Comment:
Maybe add a docstring?

Reply:
I added a docstring and cleaned up the class.